src/Fortify/Components/RegisterForm.php

```diff
@@ -76,22 +76,24 @@ public function canSubmit(): bool
 
     public function updated(string $propertyName): void
```

Comment:
@alexbarnsley I think we can use the `$value` param, so we don't need to `$value = $this->{$propertyName};`... What do you think?

```suggestion
    public function updated(string $propertyName, mixed $propertyValue): void
```

Reply:
can take a look/test it @leMaur :+1: